middleware.js

```diff
@@ -52,6 +52,20 @@ function pickLanguage (req, res, next) {
 }
 
 
+async function recordVisitFromEmail (req, res, next) {
+  if (req.query.utm_source && req.query.utm_source !== "fx-monitor-email") {
```

Comment:
Depending on how we want FxA to categorize these events, we may want to just explicitly set `utm_source` here to `fx-monitor`.

Reply:
I'll update to be `utm_source=fx-monitor` but that will affect all our other uses of `appendUtmParams`. So we're basically converting all our email traffic from:
* `utm_source=fx-monitor-email&utm_medium=email`
to: 
* `utm_source=fx-monitor&utm_medium=email`

Which I think makes sense. We just may want to make a note in our GA reports about this change-over.